fdbserver/Status.actor.cpp

```diff
@@ -2638,18 +2629,17 @@ static JsonBuilderObject tlogFetcher(int* logFaultTolerance,
 			remote_log_fault_tolerance = tLogSet.tLogReplicationFactor - 1 - failedLogs;
 		}
 	}
-	if (minFaultTolerance == 1000) {
-		// just in case we do not have any tlog sets
-		minFaultTolerance = 0;
-	}
-	if (localSetsWithNonNegativeFaultTolerance > 1) {
-		minFaultTolerance++;
-	}
-	*logFaultTolerance = std::min(*logFaultTolerance, minFaultTolerance);
+	
+	FaultTolerance = FaultTolerance.value_or(0);
```

Comment:
Are Optional really necessary?  why not to use a simple int with the 0 initial value?